index.html

```diff
@@ -1,14 +1,23 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.css">
+    <link rel="stylesheet" href="https://unsplash.com/search/photos/bicycle">
     <link rel="stylesheet" href="main.css">
     <title></title>
-  </head>
-  <body>
-    <img src="assignments/images/bike logo clipart.png" alt="">
-    <h1>Baron Crek Ccling</h1>
```

Comment:
I'm not sure your master index file is the most current one since you have the typos from an older file.